snorkel/classification/models/advanced/advanced_classifier.py

```diff
@@ -119,7 +119,7 @@ def forward(  # type: ignore
         X_dict = move_to_device(X_dict, self.config["device"])
 
         outputs: Dict[str, Mapping[Union[str, int], Any]] = {}
-        outputs["_input_"] = X_dict
+        outputs["_input_"] = X_dict  # type: ignore
```

Comment:
What's the issue here?

Reply:
The type of outputs is: Dict[str, Mapping[Union[str, int], Any]]
The type of X_dict is: Mapping[str, Any]
 
outputs["__input__": X_dict] is unhappy that X_dict cannot possibly be Mapping[int, Any].

Reply:
Try Dict[str, Union[Mapping[int, Any], Mapping[str, Any]]]